src/fu-util-common.c

```diff
@@ -670,6 +670,24 @@ fu_util_release_get_name (FwupdRelease *release)
 			 * at system bootup */
 			return g_strdup_printf (_("%s CPU Microcode Update"), name);
 		}
+		if (g_strcmp0 (cat, "X-Battery") == 0) {
+			/* TRANSLATORS: battery refers to the system laptop power source */
```

Comment:
I think that many things have BMUs, they don't have to be laptops :)

Reply:
Agreed, thanks.